libraries/Microsoft.Bot.Builder/ActivityHandler.cs

```diff
@@ -179,14 +179,14 @@ protected virtual Task OnMessageActivityAsync(ITurnContext<IMessageActivity> tur
         /// <seealso cref="OnMembersRemovedAsync(IList{ChannelAccount}, ITurnContext{IConversationUpdateActivity}, CancellationToken)"/>
         protected virtual Task OnConversationUpdateActivityAsync(ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
         {
-            if (turnContext.Activity.MembersAdded != null)
+            if (turnContext.Activity.MembersAdded?.Count > 0)
```

Comment:
Couldnt be using .Any() for these cases, there are a few more, dont remember in the other PRs?

Reply:
I wasn't sure if Any() was applicable to possible null variables, but they shouldn't be null, so I fixed them.
Thanks!